CHANGELOG.md

```diff
@@ -0,0 +1,29 @@
+# Changelog
```

Comment:
Added a changelog to be more explicit with the state of the course example

Reply:
Ohh that's a great idea!